src/module/displayList.js

```diff
@@ -2,8 +2,15 @@ import fetchList from './fetchList.js';
 
 const displayList = async () => {
   const url = 'https://www.themealdb.com/api/json/v1/1/categories.php';
+  const urli = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/gXvWpQoPOY3Cj1qr1TRg/likes';
 
   const category = await fetchList(url);
+  const likesdata = await fetchList(urli);
+
+  function displayLike(id) {
+    const itemsID = likesdata.filter((element) => element.item_id === id);
+    return itemsID[0].likes ? itemsID[0].likes : 0;
+  }
```

Comment:
- [ ] Please I suggest using the `arrow function` instead. As per requirement, all `functions` are to be converted into `arrow functions`. This is because they have shorter syntax than regular function expressions, and implicit return statements, and increases readability.